python/paddle/fluid/tests/unittests/test_conv2d_int8_mkldnn_op.py

```diff
@@ -203,26 +220,54 @@ def init_group(self):
         self.groups = 3
 
 
-#--------------------test conv2d s8 in and s8 out--------------------
-
-
-def create_test_int8_class(parent):
+def create_test_int8_class(parent, input_dt, fuse_relu):
     class TestInt8Case(parent):
         def init_dtype(self):
-            self.srctype = np.int8
-            self.dsttype = np.int8
+            if input_dt == np.uint8:
```

Comment:
please simplify line247-270 like `test_pool2d_int8_mkldnn_op.py`
```
#--------------------test conv2d s8 in and u8 out--------------------
class TestS8U8Case(parent):
   xxx
#--------------------test conv2d s8 in and s8 out--------------------
class TestS8S8Case(parent):
   xxx
#--------------------test conv2d u8 in and s8 out--------------------
class TestU8S8Case(parent):
   xxx
```

Reply:
it's not simple s8 and u8 case, residual should be considered to test cases,  “def create_test_int8_class(parent, input_dt, fuse_relu, fuse_residual):” is a good function structure, can we keep it?

Reply:
```
def init_data_typeXXX(self, input_dt, fuse_relu, fuse_residual): # please rename init_data_typeXXX.
          xxx
def create_test_int8_class(parent):
    #--------------------test conv2d s8 in and u8 out--------------------
    class TestS8U8Case(parent):
         def init_data_type(self):
              init_data_typeXXX(self, np.int8, True)
     ....
```

Reply:
done.